src/client/WindshaftRequestHelper.js

```diff
@@ -0,0 +1,97 @@
+import { version } from '../../package';
+import CartoMapsAPIError, { CartoMapsAPIErrorTypes } from '../errors/carto-maps-api-error';
+
+const REQUEST_GET_MAX_URL_LENGTH = 2048;
+
+export default class WinshaftRequestHelper {
+    constructor (conf, mapConfig) {
+        this._conf = conf;
+        this._mapConfig = mapConfig;
+
+        this._auth = this._encodeParameter('api_key', this._conf.apiKey);
+        this._client = this._encodeParameter('client', `vl-${version}`);
+    }
+
+    async getMapRequest () {
```

Comment:
I'd name it `getLayerGroup` cause we're returning a layergroup, not a request.

Reply:
Totally 👍